docs/charliecloud.rst

```diff
@@ -0,0 +1,68 @@
+.. _charliecloud-page:
+
+************************
+Charliecloud containers
+************************
+
+Nextflow integration with `Charliecloud <https://hpc.github.io/charliecloud>`_ technology allows you to write self-contained
+and truly reproducible computational pipelines.
+
+By using this feature any process in a Nextflow script can be transparently executed into a Podman container. This may
+be extremely useful to package the binary dependencies of a script into a standard and portable format that can be 
+executed on any platform supporting the Podman engine.
+
+.. note::
+    This feature requires Nextflow version ``20.01.0`` or later.
```

Comment:
Can you please change this to version `20.12.0-edge`  ?

Reply:
Done, I also updated the docs.